pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -789,6 +789,8 @@ func (m *migrationMonitor) startMonitor() {
 	logger := log.Log.Object(vmi)
 	defer func() {
 		m.l.migrateInfoStats = &stats.DomainJobInfo{
+			MemoryBps:     0,
+			DiskBps:       0,
```

Comment:
Defining zeros here is redundant since it's `uint64`'s default anyways.
You can just go with:
```go
m.l.migrateInfoStats = &stats.DomainJobInfo{}
```

Reply:
cool, will fix